ui/ozone/platform/wayland/wayland_input_method_context.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/ozone/platform/wayland/wayland_input_method_context.h"
+
+#include "base/bind.h"
+#include "base/memory/ptr_util.h"
+#include "base/strings/string_util.h"
+#include "base/strings/utf_string_conversions.h"
+#include "ui/events/event.h"
+#include "ui/ozone/platform/wayland/wayland_connection.h"
+#include "ui/ozone/platform/wayland/zwp_text_input_wrapper_v1.h"
+
+namespace ui {
+
+WaylandInputMethodContext::WaylandInputMethodContext(
+    WaylandConnection* connection)
+    : connection_(connection), text_input_(nullptr) {
+  use_ozone_wayland_vkb_ = getenv("USE_OZONE_WAYLAND_VKB");
```

Comment:
Better using a command line switch. It should be possible to just add one in ozone_switches. Maybe a set of --enable-ozone-ime/--disable-ozone-ime, that would be enabled/disabled by default depending on the platform (i.e. I guess it should be disabled in all ozone platforms right now?). Not sure about ChromeOS default behavior in this case, as the IME they are exposing is not going through ozone right?